src/App.tsx

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+
+type State = {
+  x: string;
```

Comment:
```suggestion
  pressedKey: string;
```
![image](https://user-images.githubusercontent.com/96409563/226717512-104e4c46-0cb2-4b26-9374-5f1d1fdfd7c2.png)